docs/notebooks/regional_frequency_analysis.ipynb

```diff
@@ -0,0 +1,970 @@
+{
```

Comment:
<p>J'ajouterais un bout de phrase pour indiquer ce que ça signifie.</p><p> </p><blockquote>, which means that the first 3 components are independent enough to be used for the rest of our analysis.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>